test/unit/walk-manifest.spec.js

```diff
@@ -9,6 +9,44 @@ const walker = require('../../src/walk-manifest');
 
 const TEST_URL = 'http://manifest-list-test.com';
 
+const countResources = function({
```

Comment:
reusable helper, we did something like this in almost all of the tests. Now we do it in all of them.

Reply:
Most of the changes in this file are switching the tests to use this, and returning the promise that we already have instead of using `done`